pkg/identity/cache_test.go

```diff
@@ -34,6 +34,18 @@ func (s *IdentityTestSuite) TestLookupReservedIdentity(c *C) {
 	identity = LookupIdentity(labels.NewLabelsFromModel([]string{"reserved:world"}))
 	c.Assert(identity, Not(IsNil))
 	c.Assert(identity.ID, Equals, worldID)
+
+	initWellKnownIdentities()
```

Comment:
Will we still retain the concept of "fixed-identity"? It seems like we will replace it with this PR?

Reply:
Yes, they are part of the API and useful. We are just not using it for EKS anymore.